Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -449,8 +449,6 @@ namespace AzToolsFramework
 
     static void InitializeTranslationLookup(EntityIdManipulators& entityIdManipulators)
     {
-        AZ_PROFILE_FUNCTION(AzToolsFramework);
```

Comment:
Is this not still useful? Or is it too much noise in the profiler?

Reply:
Yeah it's just noise right now and not something we're worried about actively tracking